gp-templates/helper-functions.php

```diff
@@ -211,8 +211,38 @@ function textareas( $entry, $permissions, $index = 0 ) {
 }
 
 function display_status( $status ) {
+	$status_translated = [
+    'current' => _x( 'current', 'Single Status', 'glotpress' ),
+	];
+	$status_waiting = [
+		'waiting' => _x( 'waiting', 'Single Status', 'glotpress' ),
+	];
+	$status_fuzzy = [
+		'fuzzy' => _x( 'fuzzy', 'Single Status', 'glotpress' ),
+	];
+	$status_old = [
+		'old' => _x( 'old', 'Single Status', 'glotpress' ),
+	];
+	$status_rejected = [
+		'rejected' => _x( 'rejected', 'Single Status', 'glotpress' ),
+	];
```

Comment:
All the `$status_xx` arrays should be combined into one `$status_labels` array.

Reply:
Done in https://github.com/GlotPress/GlotPress-WP/pull/813/commits/2fbaef31ed66b3cdf24de835954ffa71444e43a0

Reply:
Done in https://github.com/GlotPress/GlotPress-WP/pull/813/commits/199047666380192c7006f391179fdf158989242a